Makefile.am

```diff
@@ -51,6 +51,20 @@ dist_noinst_DATA= \
 	configs.signatures \
 	Dockerfile \
 	netdata.spec \
+	os-templates/conf.d/linux/health.d/cpu.conf \
```

Comment:
This should go at least into 
os-templates/conf.d/Makefile.am
I am not sure if we should put it deeper into the path if possible.